templates/bundles/DemosPlanProcedureBundle/DemosPlanProcedure/public_index_globalnewslist.html.twig

```diff
@@ -1,90 +1,93 @@
 <section class="{{ 'o-page__padded--spaced o-page__news u-pt'|prefixClass }}">
-
-    <header>
-        <h2 class="{{ 'layout__item u-1-of-2 u-pl-0'|prefixClass }}">
-            {{ "news.announcements"|trans }}
-        </h2>
-        <a class="{{ 'layout__item u-1-of-2 text--right u-mb-0_5 u-pl-0'|prefixClass }}" href="{{ path('DemosPlan_globalnews_news') }}">
-            <span>
-                {{ "news.see.all"|trans }}
-            </span>
-            <i class="{{ 'fa fa-angle-right'|prefixClass }}" aria-hidden="true"></i>
-        </a>
-    </header>
-
-    <div class="{{ 'layout'|prefixClass }}">
-        {% set cols = templateVars.list.newslist|length %}
-        {% set classes = 'u-1-of-' ~ cols ~ ' layout__item u-1-of-1-palm u-mb-0_5-palm o-page__news-item' %}
-        {% for newsItem in templateVars.list.newslist %}
-
-            <div class="{{ classes|prefixClass }}">
-
-                {# title #}
-                <h3 class="{{ 'font-size-h3 u-mb-0'|prefixClass }}">
-                    <a title="{{ newsItem.title|default( "news"|trans ) }} lesen" href="{{ path('DemosPlan_globalnews_news_detail',{'newsID':newsItem.ident}) }}">
-                      {{ newsItem.title }}
-                   </a>
-                </h3>
-
-                {# meta #}
-                <div class="{{ 'cf u-mb-0_25'|prefixClass }}">
-
-                    {# creation date #}
-                    {% if newsItem.createDate is defined %}
-                        <span class="{{ 'font-size-smaller'|prefixClass }}">
-                            {{ newsItem.createDate|default()|dplanDate }}
-                        </span>
-                    {% endif %}
-
-                    {# label 'press' if press category found #}
-                    {% if newsItem.categories is defined and newsItem.categories[0] is defined %}
-                        {% if 'press' == newsItem.categories[0].name %}
-                            <span class="{{ 'font-size-smaller u-ml-0_25'|prefixClass }}">
-                               Pressebericht
+    {% if templateVars.list.newslist is not empty %}
```

Comment:
Can't we wrap the `if` around the `include` of this template (`public_index.html.twig:34`)? With the current solution, an empty `section` is displayed.

Reply:
Ah, I see. I couldn't find a better solution either, and in the end having an empty section does not seem too problematic in terms of accessibility, so let's leave it like this.